src/pages/experiments/[experimentId]/data-processing/configure-embedding/components/EmbeddingPreview.jsx

```diff
@@ -39,20 +34,22 @@ import LabelsDesign from '../../../plots-and-tables/components/LabelsDesign';
 
 const { Panel } = Collapse;
 
-const EmbeddingPreview = () => {
+const EmbeddingPreview = (props) => {
```

Comment:
it will be worth looking at Agi's PR and chatting with him - he has done lots of changes to this component in his PR too.

Reply:
Latest changes pushed to #95. I don't think there'll be additional major changes, just unit tests left